app/Compiler/Twig/templates/baseRecord.twig

```diff
@@ -26,6 +28,10 @@ abstract class BaseRecord implements JsonSerializable
         return json_decode(json_encode($mixed), true);
     }
 
+    /**
+     * @param array[string]mixed $array - Array holding arbitrary data to be decoded into this object.
```

Comment:
Does `array[string]mixed` mean something in PHPDoc or is that a typo?

Reply:
That was my misguided attempt to type nested arrays from the other day that carried over to this. Going to just keep it as "array"